### DISCUSSION TITLE
Can I name a project after this one?

### DISCUSSION THREAD

**Discussion by ghost** [hyprwm/Hyprland#370] [Q&A] [answered]
Basically, I'm considering trying to write a bar that works better with Hyprland than other bars. If I end up starting the project, can I name it Hyprbar? I just want to get permission before I do it.

**Comment by vaxerski** [comment, marked as answer]
Even though I cannot say no, as we do not have a trademark on "Hypr" or "Hyprland", I'd recommend you to find a name for it that doesn't really blend into the hypr "ecosystem" too much, as nobody from the Hypr team will be working on it. Unless of course you want to start it under the hyprwm organization, but that would require you to follow our standards.

**Reply by ghost** [reply]
Good enough of a reason for me not to use it. Thank you for your honesty! As a side note, what would those standards be? Even though I'm not going to start the project under the Hyprwm org, I want to follow the guidelines so that I write good code.

**Reply by vaxerski** [reply]
The standards regard code styling and architecture. You can get a grasp of them by reading Hyprland's code.

If you want a detailed explanation, dm me on discord.
